.circleci/config.yml

```diff
@@ -219,7 +197,7 @@ jobs:
     working_directory: /home/node/dlp-samples
   system_tests:
     docker:
-      - image: 'node:8'
+      - image: 'node:10'
```

Comment:
I wouldn't change this just yet.  8 is still latest LTS.

Reply:
sounds good - will revert those back to node8